ydb/core/blobstorage/vdisk/skeleton/blobstorage_skeleton.cpp

```diff
@@ -2329,6 +2329,7 @@ namespace NKikimr {
             // generation independent self VDisk Id
             auto genIndSelfVDiskId = SelfVDiskId;
             genIndSelfVDiskId.GroupGeneration = -1;
+            IFaceMonGroup->MinHugeBlobInBytes() = Config->MinHugeBlobInBytes;
```

Comment:
This value is not the actual size of a huge blob; the actual size is aligned to the append block size and can be found by the strange name MinREALHugeBlobSize (inside HugeBlobCtx).